libsolidity/parsing/Scanner.cpp

```diff
@@ -82,13 +82,9 @@ bool isWhiteSpace(char c)
 {
 	return c == ' ' || c == '\n' || c == '\t' || c == '\r';
 }
-bool isIdentifierStart(char c)
```

Comment:
They behave differently, but it's totally achievable to do what that function is doing with the other two functions available that make up isIdentifierStart.